README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
Not a huge one, but I was trying out the Semaphore badge, and they don't have one for RDOC, but they do for markdown.  If there is a way to do this in RDOC, then maybe we could try that.